src/lib/datadog/index.tsx

```diff
@@ -28,11 +28,10 @@ const datadogScriptBody = `(function(h,o,u,n,d) {
      env: '${ENV}', 
      ${COMMIT_SHA ? `version: '${COMMIT_SHA}',` : ''}
      sampleRate: 100,
-     premiumSampleRate: 100,
+     premiumSampleRate: 0,
```

Comment:
Along with `premiumSampleRate`, `sessionReplaySampleRate` needs to be added and set to `0`.

Reply:
oh it looks like `replaySampleRate` might be good to add as well. even though it is for DD versions `4.10.1` and before

Reply:
Ah oops great find - looks like [this changed in a recent version](https://docs.datadoghq.com/real_user_monitoring/guide/sampling-browser-plans/#setup), and seems like `premiumSampleRate` is now deprecated?

I'm unclear how versioning happens - seems like since our script URL is `https://www.datadoghq-browser-agent.com/datadog-rum-v4.js` we get automatically bumped to the latest `4.x` version as it arrives? I'll leave both in place at `0` just in case 👍